README.md

```diff
@@ -522,6 +522,16 @@ To run tests in IntelliJ, go to your preferences, click plugins, and make sure y
 
 Once you have JUnit installed, go to the test file you want to run, right click on the test file, and select `Run` or `Debug` to run or debug the tests. To run or debug on a specific test class or method, open the file, and test running options should appear as green triangles pointing right on the side bar. Clicking that will open a dialog allowing you to run that specific class or method.
 
+### Debugging a Running API in IntelliJ
+
+Choose `Edit Configurations...` from the dropdown at the left side of the top the menu bar
+and choose `Templates` -> `Remote`.  Click `Create configuration` in the top right.  This
+will create a Remote Configuration which you can name how you like.  Edit this configuration
+to choose port *8001*.
```

Comment:
This has bitten me several times.  It's not the IntelliJ default of 5005.

Alt: change the port we use for this.

But that's a bigger task and I'm not sure of the implications.  This will at least capture our current reality.

Reply:
Maybe 'Ensure this configuration is port *8001*'? Since its a general debugging section, I don't think we should say edit here specifically.

Reply:
updated